au4/src/playback/CMakeLists.txt

```diff
@@ -68,7 +68,8 @@ set(MODULE_SRC
 
     )
 
-set(LINK_LIB
+set(MODULE_LINK
+    projectscene
```

Comment:
This raises doubts, it seems that playback should not depend on the projectscene.
Options:
* Move the playtoolbar to the projectscene (favorite)
* Move this component to a common place, maybe something like commonscene, or even to uicomponents.

I'm not calling for this to be fixed right now, but it's technical debt. If it's not fixed, there will be a cyclical dependency.

Reply:
https://github.com/audacity/audacity/issues/6872